bin/ember-template-lint.js

```diff
@@ -45,6 +46,18 @@ function expandFileGlobs(filePatterns, ignorePattern) {
   let result = new Set();
 
   filePatterns.forEach((pattern) => {
+    // if we are passed items that are already globbed before we should not crawl them as directories
+    if (pattern.slice(-4) === '.hbs' && pattern.indexOf('*') === -1) {
```

Comment:
I think there is a utility for this :thinking: 

Something something "has magic"....

Reply:
I used the is-valid-glob module from micromatch.